ODS AMI E2E quickstarter prov app fails due to no nexus equal true

This also affect branches `master` and `4.x`

logs:
```
[2022-03-08T22:31:42.918Z] [0;32m    amazon-ebs: === CONT  TestQuickstarter/ods-provisioning-app[0m
[2022-03-08T22:31:42.918Z] [0;32m    amazon-ebs: Run step opendevstack/ods-core#1 (provision) of quickstarter ods-quickstarters/ods-provisioning-app ...[0m
[2022-03-08T22:31:43.174Z] [0;32m    amazon-ebs: -- starting cleanup for component: provapp[0m
[2022-03-08T22:31:43.174Z] [0;32m    amazon-ebs: -- delete resources labelled with: app=unitt-provapp[0m
[2022-03-08T22:31:43.430Z] [0;32m    amazon-ebs: -- cleaned up resources with label: app=unitt-provapp[0m
[2022-03-08T22:31:43.430Z] [0;32m    amazon-ebs: -- starting build for: ods-provisioning-app/Jenkinsfile in project: opendevstack[0m
....
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: + ./gradlew clean build --stacktrace --no-daemon[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Picked up JAVA_TOOL_OPTIONS: -XX:+UnlockExperimentalVMOptions -Dsun.zip.disableMemoryMapping=true[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Downloading https://services.gradle.org/distributions/gradle-6.7.1-bin.zip[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:  > git rev-parse --verify HEAD # timeout=10[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Resetting working tree[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:  > git reset --hard # timeout=10[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:  > git clean -fdx # timeout=10[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: .........10%..........20%..........30%..........40%..........50%.........60%..........70%..........80%..........90%..........100%[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Welcome to Gradle 6.7.1![0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Here are the highlights of this release:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:  - File system watching is ready for production use[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:  - Declare the version of Java your build requires[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:  - Java 15 support[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: For more details see https://docs.gradle.org/6.7.1/release-notes.html[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: WARNING: An illegal reflective access operation has occurred[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass (file:/home/jenkins/.gradle/wrapper/dists/gradle-6.7.1-bin/bwlcbys1h7rz3272sye1xwiv6/gradle-6.7.1/lib/groovy-all-1.3-2.5.12.jar) to method java.lang.Object.finalize()[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: WARNING: All illegal access operations will be denied in a future release[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: FAILURE: Build failed with an exception.[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: * Where:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Build file '/tmp/workspace/unitt-cd/unitt-cd-run-unitt-provapp-unitt-master/build.gradle' line: 9[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: * What went wrong:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: A problem occurred evaluating root project 'unitt-cd-run-unitt-provapp-unitt-master'.[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: > property 'no_nexus' is set to false, but neither 'nexus_url', 'nexus_user' nor 'nexus_pw' is configured. You can do so by e.g. creating a gradle.properties file in your 'GRADLE_USER_HOME' (by default '~/.gradle') folder and setting the nexus properties there.[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: * Try:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: Run with --info or --debug option to get more log output. Run with --scan to get full insights.[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: * Exception is:[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: org.gradle.api.GradleScriptException: A problem occurred evaluating root project 'unitt-cd-run-unitt-provapp-unitt-master'.[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: 	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:93)[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl.apply(DefaultScriptPluginFactory.java:111)[0m
[2022-03-08T22:34:46.060Z] [0;32m    amazon-ebs: 	at org.gradle.configuration.BuildOperationScriptPlugin$1.run(BuildOperationScriptPlugin.java:65)[0m
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,7 @@
 - Fix problem assigning admin permissions to bitbucket repositories ([#700](https://github.com/opendevstack/ods-provisioning-app/pull/700))
 - Update OpenShift client to use fabric8 OpenShift client (OpenShift 4 compatible) ([#720](https://github.com/opendevstack/ods-provisioning-app/pull/720))
 - Disable openshift service adapter by default ([#721](https://github.com/opendevstack/ods-provisioning-app/pull/721))
+- ODS AMI E2E quickstarter prov app fails due to no nexus equal false ([#730](https://github.com/opendevstack/ods-provisioning-app/pull/730))
 
 ### Added
 

--- a/build.gradle
+++ b/build.gradle
@@ -1,10 +1,10 @@
 buildscript {
     ext {
         // set gradle properties regarding nexus
-        nexus_url = project.findProperty('nexus_url') ?: System.getenv('NEXUS_URL')
+        nexus_url = project.findProperty('nexus_url') ?: System.getenv('NEXUS_HOST')
         nexus_user = project.findProperty('nexus_user') ?: System.getenv('NEXUS_USERNAME')
         nexus_pw = project.findProperty('nexus_pw') ?: System.getenv('NEXUS_PASSWORD')
-        no_nexus = project.findProperty('no_nexus') ?: System.getenv('NO_NEXUS') ?: true
+        no_nexus = project.findProperty('no_nexus') ?: System.getenv('NO_NEXUS') ?: false
         if (!no_nexus && (nexus_url == null || nexus_user == null || nexus_pw == null)) {
             throw new GradleException("property 'no_nexus' is set to false, but neither " +
                 "'nexus_url', 'nexus_user' nor 'nexus_pw' is configured. You can do so " +

--- a/build.gradle
+++ b/build.gradle
@@ -4,7 +4,7 @@ buildscript {
         nexus_url = project.findProperty('nexus_url') ?: System.getenv('NEXUS_URL')
         nexus_user = project.findProperty('nexus_user') ?: System.getenv('NEXUS_USERNAME')
         nexus_pw = project.findProperty('nexus_pw') ?: System.getenv('NEXUS_PASSWORD')
-        no_nexus = project.findProperty('no_nexus') ?: System.getenv('NO_NEXUS') ?: false
+        no_nexus = project.findProperty('no_nexus') ?: System.getenv('NO_NEXUS') ?: true
         if (!no_nexus && (nexus_url == null || nexus_user == null || nexus_pw == null)) {
             throw new GradleException("property 'no_nexus' is set to false, but neither " +
                 "'nexus_url', 'nexus_user' nor 'nexus_pw' is configured. You can do so " +

